src/gui/MainWindow.cc

```diff
@@ -2017,6 +2021,8 @@ void MainWindow::sendToOctoPrint()
   const QString fileFormat = QString::fromStdString(Settings::Settings::octoPrintFileFormat.value());
   FileFormat exportFileFormat{FileFormat::STL};
   if (fileFormat == "OFF") {
+    exportFileFormat = FileFormat::OBJ;
+  } else if (fileFormat == "OFF") {
     exportFileFormat = FileFormat::OFF;
```

Comment:
Typo, one shoudl be `OBJ`.

Reply:
typo fixed
tests added
see 4 commits below